sponsorship.html

```diff
@@ -294,37 +339,35 @@
         </div>
     </div>
 
-    <div class="container_1">
-
-        <header>
-            <div class="logos">
-                <img src="https://seeklogo.com/images/C/cleveland-state-university-logo-F988F5A8C8-seeklogo.com.png"
-                    alt="Cleveland State University Logo">
-            </div>
-            <div class="title">
-                <h1>Formula Vikes Racing</h1>
-                <h2>Sponsorship Flyer</h2>
-            </div>
-            <div class="logos">
-                <img src="https://upload.wikimedia.org/wikipedia/commons/e/ed/SAE_International_logo.svg"
-                    alt="SAE International Logo">
-            </div>
-        </header>
-
-        <section class="about-section">
-            <h3>About Formula SAE:</h3>
-            <p>Formula SAE (Society of Automotive Engineers) is a collegiate design competition that challenges students
-                with real-world engineering. Students are tasked with designing and fabricating a formula-style car to
-                compete against other formula-style cars.</p>
-        </section>
+    <div class="packet">
+        <a href="https://drive.google.com/file/d/1avAfzvM-uNSiDVxJ-WUDdqQ9AQvuIeOS/view?usp=sharing">Sponsorship Packet</a>
```

Comment:
Sponsor packet should be hosted on the github as a PDF. If a unauthorized user tries to access the google drive they will be denied